src/hooks/useEditorEventCallback.ts

```diff
@@ -31,12 +46,8 @@ export function useEditorEventCallback<T extends unknown[], R>(
 
   return useCallback(
     (...args: T) => {
-      // It's not actually possible for an event handler to run
-      // while view is null, since view is only ever set to
-      // null in a layout effect that then immediately triggers
-      // a re-render which sets view to a new EditorView
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      return ref.current(view!, ...args);
+      assertIsReactEditorView(view);
+      return ref.current(view, ...args);
```

Comment:
This is the only user-facing change, I think. It did affect our demo but I don't think it's likely to affect many. We can document it in release notes. I'm not sure whether to call it breaking because it was not a safe assumption before that the view was non-null. One could have used this hook improperly before and found that it was invoked with a null view.